index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!--poppins font link-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,300;0,600;0,800;1,200;1,600&display=swap"
+      rel="stylesheet">
+    <!--fontawsome link-->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <!--styles.css link-->
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+  </head>
+
+  <body>
+    <header>
+      <div class="logo-container">
+        <div class="logo">
+          <a href="index.html">Anold</a>
+        </div>
+          <div class="bars">
+              <i class="fa fa-bars"></i>
+          </div>
```

Comment:
- [ ] Good job trying to use the `<header>`  tag as suggested by the previous reviewer. Please note that only your logo and the hamburger should be wrapped in the `<header>` tag and the hamburger wrapped in the `<nav>` tag within the header to male your website assessible👍🏽.

e.g
```
    <header>
        <a href="index.html">Anold</a>
        <nav class="logo-container">
             <div class="bars">
              <i class="fa fa-bars"></i>
          </div>
        </nav>
        </header>
```